src/main/java/ch/psi/jcae/impl/ChannelAccessException.java

```diff
@@ -0,0 +1,13 @@
+package ch.psi.jcae.impl;
+
+import gov.aps.jca.CAStatus;
+import java.util.concurrent.ExecutionException;
+
+/**
+ *
+ */
+class ChannelAccessException extends ExecutionException{
```

Comment:
As this exception does nothing more than transporting a String, it would be better to just use a RuntimeException